src/Smart.Design.Console/Models/Exceptions/CustomExceptions.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Please be careful, don't retain this kind of sentence that will be wrong when we add our own code.